app/Photo.php

```diff
@@ -192,7 +192,7 @@ public function prepareData()
 		$photo['license'] = Configs::get_value('default_license'); // default
 
 		// shutter speed needs to be processed. It is stored as a string `a/b s`
-		if ($photo['shutter'] != '' && substr($photo['shutter'], 0, 2) != '1/') {
+		if ($photo['shutter'] != '' && substr($photo['shutter'], 0, 2) != '1/' && substr($photo['shutter'], 1, 2) != '/0') {
```

Comment:
I wonder if this new test would be better after L197. I'm not sure either way, but it would catch `15/0` and allow `1/02`. Neither of these should happen, but...

Reply:
fixed.

Reply:
@d7415  @ildyria thanks for the fix. I've been busy on other works these days. sorry.

Reply:
@ttys3 it is always a pleasure to have your PR. :+1: